src/app/screens/ledger/verifyLedgerAccountAddress/index.tsx

```diff
@@ -175,17 +175,17 @@ function VerifyLedger(): JSX.Element {
 
   const importBtcAccounts = async (showAddress: boolean, masterFingerPrint?: string) => {
     const transport = await Transport.create();
-    const addressIndex = ledgerAccountsList.filter(
-      (account) => account.masterPubKey === (masterFingerPrint || masterPubKey),
-    ).findIndex((account) => account.id === selectedAccount?.id);
+    const addressIndex = ledgerAccountsList
+      .filter((account) => account.masterPubKey === (masterFingerPrint || masterPubKey))
+      .findIndex((account) => account.id === selectedAccount?.id);
 
     if (isBitcoinSelected) {
       try {
         await importNativeSegwitAccountFromLedger(
           transport,
           network.type,
-          0,
           addressIndex,
+          0,
           showAddress,
```

Comment:
was this a bug?

i'd propose we open a linear task to refactor xverse-core ledger/index.ts to use Object named params

Reply:
I think it works fine without this param place replacement. Created the linear task for Object named params refactoring: https://linear.app/xverseapp/issue/ENG-2537/refactor-xverse-core-ledgerindexts